src/components/BarChart/BarChart-redux.tsx

```diff
@@ -21,20 +22,35 @@ const BarChartRedux = () => {
 
     const fetchEthereumTransactionsSummed = (transactionsSummedData: TransactionsSummedData): void => {
         dispatch({
-            type: transactionsActions.ETHEREUM_FETCH_TRANSACTIONS_SUMMED,
+            type: transactionsActionsSummed.ETHEREUM_FETCH_TRANSACTIONS_SUMMED,
             transactionsSummedData: transactionsSummedData
         });
     };
+    
+    const fetchEthereumTransactions = (transactionsData: TransactionsData): void => {
+        dispatch({
+            type: transactionsActions.ETHEREUM_FETCH_TRANSACTIONS,
+            transactionsData: transactionsData
+        });
+    };
```

Comment:
Why is this here? Ethereum transactions are not used in Barchart. Move it to TransactionsList component.